test/e2e/specs/import_wallet.spec.js

```diff
@@ -0,0 +1,147 @@
+import {
+  checksumAddress,
+  v3password,
+  privateKey,
+  mnemonic,
+} from '../fixtures/accounts';
+
+describe('Import Wallet Page', () => {
+  beforeEach(() => {
+    cy.getInitialData();
+    cy.visit('#/import');
+    cy.waitPageLoad();
+  });
+
+  it('should import wallet from private key', () => {
+    cy.contains('Private Key').click();
+    cy.get('[data-test=input-private-key] input').type(privateKey);
+    cy.get('[data-test=input-wallet-password] input').type(v3password);
+    cy.get('[data-test=submit-import]').click();
+    cy.get('[data-test=address-card]', { timeout: 25000 })
+      .contains(checksumAddress)
+      .should('exist');
+    cy.get('[data-test=nav-sidebar-menu]')
+      .contains('Send')
+      .should('exist');
+  });
+
+  it('should import wallet from public key', () => {
+    cy.contains('Address').click();
+    cy.get('[data-test=input-public-key]').type(checksumAddress);
+    cy.get('[data-test=submit-import]').click();
+    cy.get('[data-test=address-card]', { timeout: 25000 })
+      .contains(checksumAddress)
+      .should('exist');
+    // FIXME after bug fix
+    // cy.get('[data-test=nav-sidebar-menu]')
+    //   .contains('Send')
+    //   .should('not.exist');
+    // cy.get('[data-test=nav-sidebar-menu]')
+    //   .contains('Tools')
+    //   .should('not.exist');
+  });
+
+  it('should import wallet from seed phrase', () => {
+    cy.contains('Seed Phrase').click();
+    cy.get('[data-test=input-seed-phrase]').type(mnemonic);
+    cy.get('[data-test=input-wallet-password] input').type(v3password);
+    cy.get('[data-test=submit-import]').click();
+    cy.get('[data-test=address-card]', { timeout: 25000 })
+      .contains(checksumAddress)
+      .should('exist');
```

Comment:
[The same](https://github.com/endpass/endpass-wallet/pull/244/files#diff-2f46c069e03fb61aee6c10f29498562aR25)